test/minikube/minikube_base_database_test.go

```diff
@@ -142,41 +142,44 @@ func verifyEngineAltAddress(t *testing.T, namespaceName string, admin0 string, e
 }
 
 func backupDatabase(t *testing.T, namespaceName string, podName string, databaseName string, options *helm.Options) {
-	randomSuffix := strings.ToLower(random.UniqueId())
+	// NTJ- currently database chart automatically starts the hotcopy Job
```

Comment:
if the logic has changed, please fix the test to represent the new logic. Do not leave commented-out code around.

Reply:
Done in a previous push :)